app/assets/javascripts/browsered/web-payments/apple-pay.js

```diff
@@ -70,5 +70,16 @@ module.exports = () => {
     })
   }
 
+  session.onshippingcontactselected = function (event) {
+    console.log(event)
+    const { email } = event.shippingContact
+
+    if (rfc822Validator(email)) {
```

Comment:
could you use a ternary operator for this if statement?

Reply:
Yeah good point